docker/docker-compose.override-example.yml

```diff
@@ -30,13 +31,21 @@ services:
         medizininformatik-initiativede_reportReceive|1.1,
         medizininformatik-initiativede_sendForm|1.0
       DE_MEDIZININFORMATIK_INITIATIVE_REPORT_DIC_FHIR_SERVER_BASE_URL: http://dic2-fhir-store:8080/fhir
+      DE_MEDIZININFORMATIK_INITIATIVE_REPORT_DIC_FHIR_SERVER_OAUTH2_ISSUER_URL: https://keycloak:8443/realms/blaze/protocol/openid-connect/token
+      DE_MEDIZININFORMATIK_INITIATIVE_REPORT_DIC_FHIR_SERVER_OAUTH2_TRUST_CERTIFICATES: /run/secrets/keycloak_trust_certificates.pem
+      #DE_MEDIZININFORMATIK_INITIATIVE_REPORT_DIC_FHIR_SERVER_OAUTH2_PROXY_URL: http://basic-auth-forward-proxy:8888
+      #DE_MEDIZININFORMATIK_INITIATIVE_REPORT_DIC_FHIR_SERVER_OAUTH2_PROXY_USERNAME: proxy
+      #DE_MEDIZININFORMATIK_INITIATIVE_REPORT_DIC_FHIR_SERVER_OAUTH2_PROXY_PASSWORD: password
```

Comment:
Consider configuring dic2 to use the forward-proxy for all connections by default. Could be a good way to verify all proxy related code.

Reply:
good idea, moved dic2-fhir-store-blaze in its own network, reachable only through proxy:  bddf569